rmd_files/RNAseq_report.Rmd

```diff
@@ -1900,13 +1900,19 @@ if ( !is.null(params$bcbio_rnaseq) ) {
   ##### Get patient data dir and sample file name
   dataDir <- params$dragen_rnaseq
   
-  ##### Look at countsFromAbundance parameter to change the method to generate the counts
-  txi.salmon <- tximport(paste0(dataDir, "/", list.files(dataDir, pattern=".quant.sf$")), type = "salmon", tx2gene = tx2ensembl)
+  ##### Use salmon's gene quantification output
+  if ( file.exists(paste0(dataDir, "/", list.files(dataDir, pattern=".quant.genes.sf$"))) ) {
```

Comment:
It would be safer here to handle the `list.files` outside the if calls, then store the file path in a variable to read into `counts_salmon` or `txi.salmon`.
I think we've already started handling these a bit better in the r-pkg branch but let's just go with this in the master branch for now if it saves time.

Reply:
updated in https://github.com/umccr/RNAsum/commit/ea5ae5cfbf10ad8a41a9b3c4f7136fc7a01fbb38